### DISCUSSION TITLE
How to permanently store workflows in the simple tab

### DISCUSSION THREAD

**Discussion by gymnae** [mcmonkeyprojects/SwarmUI#148] [Q&A]
I'm using the official docker image and created some ComfyUI workflows, which I added to the Simple tab through the functionality exposed in the ComfyUI tab. Every time I rebuild a fresh container from the image, these workflows are gone from the simple tab. They remain accessible in the ComfyUI tab, so they are persisted. 

Also, how to create the folder structure that is visible?

**Comment by mcmonkey4eva** [comment]
Ooh, put that on the Issues page
